lnd.go

```diff
@@ -278,3 +297,37 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+// fileExists reports whether the named file or directory exists.
+func fileExists(name string) bool {
+	if _, err := os.Stat(name); err != nil {
+		if os.IsNotExist(err) {
+			return false
+		}
+	}
+	return true
+}
+
+// genCertPair generates a key/cert pair to the paths provided.
+func genCertPair(certFile, keyFile string) error {
+	rpcsLog.Infof("Generating TLS certificates...")
+
+	org := "lnd autogenerated cert"
+	validUntil := time.Now().Add(10 * 365 * 24 * time.Hour)
```

Comment:
should `10 * 365 * 24 * time.Hour` be assigned to a constant? say `AUTOGENERATED_CERT_SECONDS`?